src/components/WidgetImage/WidgetImagePresentation.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+// FPCC
+import ImgFromId from 'components/ImgFromId'
+
+function WidgetImagePresentation({ widgetData }) {
+  const { caption, image } = widgetData.settings
+
+  const getImageElement = () =>
+    image ? <ImgFromId.Container className="w-full" id={image} /> : null
+
+  const getCaptionElement = () =>
+    caption ? (
+      <div className="flex flex-wrap">
+        <span className="font-bold">Caption:</span> {caption}
```

Comment:
Do we need this bold "Caption" label or could we just print the text? If there is no caption this might look silly, and if there is a caption it is pretty self-explanatory

Reply:
It only shows up if there is a caption but I'm happy to remove it